scripts/standardising.py

```diff
@@ -23,7 +23,7 @@
         continue
 
     s3_path = parse_path(file)
-    standardized_file_name = f"standardized_{get_file_name_from_path(s3_path.key)}"
+    standardized_file_name = f"{get_file_name_from_path(s3_path.key)}"
```

Comment:
You don't need to format the string (`f" "`) as `get_file_name_from_path()` returns a string